.github/CODEOWNERS

```diff
@@ -9,5 +9,4 @@
 # the repo. Unless a later match takes precedence,
 # @forcedotcom/pdt will be requested for
 # review when someone opens a pull request.
-*       @forcedotcom/pdt
-*       @shetzel
+*       @forcedotcom/pdt @shetzel
```

Comment:
From [About code owners](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax)